components/file-upload/src/index.js

```diff
@@ -34,9 +34,9 @@ const StyledInput = styled('input')({
 });
 
 const FileUpload = ({
-  meta, children, hint, acceptedFormats, className,
+  meta, children, hint, acceptedFormats, ...props
 }) => (
-  <Label className={className} error={meta.error}>
+  <Label error={meta.error} {...props}>
```

Comment:
we might want to have this as `<Label {...props} error={meta.error}>` so as to ensure that we don't over-write the `error` prop we're passing through here with another `error` value passed into this component

Reply:
also think that this should probably be `error={meta.touched && meta.error}`